proto/protovalidate/buf/validate/validate.proto

```diff
@@ -251,33 +245,35 @@ enum Ignore {
   //   // The uri rule applies only if the value is not the empty string.
   //   string foo = 1 [
   //     (buf.validate.field).string.uri = true,
-  //     (buf.validate.field).ignore = IGNORE_EMPTY
+  //     (buf.validate.field).ignore = IGNORE_IF_UNPOPULATED
   //   ];
   //
-  //   // IGNORE_EMPTY is equivalent to IGNORE_UNSPECIFIED in this case: the
-  //   // uri rule only applies if the field is set, including if it's set
-  //   // to the empty string.
+  //   // IGNORE_IF_UNPOPULATED is equivalent to IGNORE_UNSPECIFIED in this
+  //   // case: the uri rule only applies if the field is set, including if
+  //   // it's set to the empty string.
   //   optional string bar = 2 [
   //     (buf.validate.field).string.uri = true,
-  //     (buf.validate.field).ignore = IGNORE_EMPTY
+  //     (buf.validate.field).ignore = IGNORE_IF_UNPOPULATED
   //   ];
   //
   //   // The min_items rule only applies if the list has at least one item.
   //   repeated string baz = 3 [
   //     (buf.validate.field).repeated.min_items = 3,
-  //     (buf.validate.field).ignore = IGNORE_EMPTY
+  //     (buf.validate.field).ignore = IGNORE_IF_UNPOPULATED
   //   ];
   //
-  //   // IGNORE_EMPTY is equivalent to IGNORE_UNSPECIFIED in this case: the
-  //   // custom CEL rule applies only if the field is set, including if it's
-  //   // the "zero" value of that message.
+  //   // IGNORE_IF_UNPOPULATED is equivalent to IGNORE_UNSPECIFIED in this
+  //   // case: the custom CEL rule applies only if the field is set, including
+  //   // if it's the "zero" value of that message.
   //   SomeMessage quux = 4 [
   //     (buf.validate.field).cel = {/* ... */},
-  //     (buf.validate.field).ignore = IGNORE_EMPTY
+  //     (buf.validate.field).ignore = IGNORE_IF_UNPOPULATED
   //   ];
   // }
   // ```
-  IGNORE_EMPTY = 1;
+  IGNORE_IF_UNPOPULATED = 1;
+  // Deprecated: Use IGNORE_IF_UNPOPULATED
+  IGNORE_EMPTY = 1 [deprecated = true];
 
   // Validation is skipped if the field is unpopulated or if it is a nullable
```

Comment:
If there's a link to docs where unpopulated is defined, can we include a link to that here?

Reply:
"populated" is defined on the required constraint above. The docs define presence which we are explicitly avoiding here to prevent confusion.